org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -664,6 +664,48 @@ SsoImageCentralAmiLibrarian:
       - 'arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore'
     sessionDuration: 'PT12H'
 
+SsoKubernetesDeveloper:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.3.2/templates/SSO/aws-sso.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-kubernetes-developer'
+  StackDescription: 'SSO: Limited developer role used by kuberenetes developers'
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account:
+        - !Ref SandboxAccount
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref TBD
```

Comment:
please follow the existing pattern, this should reference a parameter and the `TBD` should be on the `Default:` of the parameter.  something like this.. `principalId: !Ref kubernetesDeveloperGroup`